docs/tutorials/onnx2tensorrt.md

```diff
@@ -25,45 +25,41 @@
 
 ```bash
 python tools/deployment/onnx2tensorrt.py \
+    ${CONFIG} \
     ${MODEL} \
     --trt-file ${TRT_FILE} \
     --input-img ${INPUT_IMAGE_PATH} \
-    --shape ${IMAGE_SHAPE} \
+    --shape ${INPUT_IMAGE_SHAPE} \
+    --min-shape ${MIN_IMAGE_SHAPE} \
     --max-shape ${MAX_IMAGE_SHAPE} \
-    --mean ${IMAGE_MEAN} \
-    --std ${IMAGE_STD} \
-    --dataset ${DATASET_NAME} \
```

Comment:
May add a clarification indicating the cleaned arguments and why they are unnecessary.

Reply:
Now we parse `normalize parameters` such as `std` and `mean` directly from config, thus they could be removed.